ci/env.sh

```diff
@@ -65,7 +65,9 @@ mkdir -p $prefetch_dir
 # Specify a wrapper where required for long-running commands
 CI_WAIT_FOR=
 # Show output of commands
-VERBOSE=true
+if [ -z $VERBOSE ]; then
+    VERBOSE=false
```

Comment:
I'm wondering if VERBOSE might be too generic a name, would it be better to be something like KABANERO_VERBOSE to avoid any potential environment clashes?

Reply:
This is what Appsody uses. Will probably try and get the same change into Appsody.